css/style.css

```diff
@@ -0,0 +1,2837 @@
+      display: none;
+  }
+
+ .menu__link {
+      display: none;
+  }
+
+  .top__wrapper-charset {
+      max-width: 100%;
+      height: auto;
+  }
+
+  .top__img {
+      max-width: 100%;
+      height: auto;
+      margin-right: 0;
+  }
+
+  .top {
+      text-align: center;
+      background: url(../img/tesla-fon.png)  no-repeat;
+      background-position: 50% 0;
+      background-size: 1000px 467px;
+  }
+
+  .button__text {
+      min-width: 120px;
+  }
+
+  .bottom {
+      background: url(../img/model-s-bottom.png) no-repeat;
+      background-position: 470px 142px;
+      background-size: 500px 100px;
+   }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 420px 150px;
+      background-size: 580px 390px ;
+  }
+  
+}
+
+@media (max-width: 1000px) {
+  .bottom {
+      background: url(../img/model-s-bottom.png) no-repeat;
+      background-position: 470px 142px;
+      background-size: 400px 100px;
+  }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 430px 150px;
+      background-size: 500px 350px ;
+  }
+
+}
+
+@media (max-width: 950px) {
+  .top__wrapper {
+      margin-left: 0;
+  }
+
+  .button__outline {
+      margin-left: 0;
+  }
+
+  .bottom {
+      background: url(../img/model-s-bottom.png) no-repeat;
+      background-position: 470px 142px;
+      background-size: 400px 100px;
+  }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 425px 150px;
+      background-size: 480px 340px ;
+  }
+
+}
+
+@media (max-width: 900px) {
+  .container {
+      max-width: 900px;
+  }
+
+  .top {
+      text-align: center;
+      background: url(../img/tesla-fon.png)  no-repeat;
+      background-position: 50% 0;
+      background-size: 900px 467px;
+  }
+
+  .charset__speed {
+      margin-left: 70px;
+  }
+
+  .button__outline {
+      margin-left: 0;
+      margin-right: 0;
+  }
+
+  .bottom {
+      background: url(../img/model-s-bottom.png) no-repeat;
+      background-position: 470px 142px;
+      background-size: 370px 100px;
+  }
+
+  .bottom__inner {
+      background: url(../img/car-ipad.png) no-repeat;
+      background-position: 430px 150px;
+      background-size: 450px 300px ;
+  }
+
+}
+
+@media (max-width: 880px) {
+  .container {
+      max-width: 880px;
+  }
+
+  .top__wrapper-charset {
+    margin-right: 0;
+  }
+
+  .charset__speed {
+      margin-left: 100px;
+  }
+
+  .top__wrapper-item {
+      margin-right: 110px;
+  }
+
+  .button__outline {
+      padding-left: 0;
+      padding-right: 20px;
+  }
+
+  .button__red {
+      padding-left: 25px;
+      padding-right: 25px;
+      margin-right: 0;
+  }
+
+  .middle {
+      max-width: 880px;
+      margin-top: 30px;
+  }
+
+  .middle__inner {
+      align-content: space-between;
+      background: url(../img/tesla-fon-ipad-2.png) no-repeat;
+      background-size: 880px auto;      
+  }
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)